### ISSUE TITLE
Fedora 35: Beta Testing

### ISSUE THREAD

**Issue by MisterGuinness** [MisterGuinness/gnome-shell-extension-sensors#51]
I just realised the [F35 beta is out](https://fedoramagazine.org/announcing-fedora-35-beta/)... well, a month ago.

I will do a test to see what surprises gnome 41 has in store.

**Comment by MisterGuinness** [comment]
So, just for fun I appended a dummy parameter to the spawn call, and shockingly that worked.
```
let mystery;
let [, output] = GLib.spawn_command_line_sync(cmd, mystery);
log('mystery='+mystery);
...
gjs[21964]: JS LOG: mystery=undefined
```
Although I have successfully tested using subprocesses and async/await, just to get it working against F35 Beta I will apply this dodgy patch and hold off to see what changes with the actual release.

I see subprocesses as a lower priority now, especially since it will require a non-trivial amount of work. There's already some old school asynchronous looking code using GLib.spawn_async_with_pipes
